nubis/puppet/files/nubis_users/nubis-users.toml

```diff
@@ -0,0 +1,12 @@
+[template]
+src = "nubis-users.tmpl"
+dest = "/etc/puppet/yaml/nubis_users/common.yaml"
+prefix = "/environment/nubis-users"
```

Comment:
So this becomes a little bit more cumbersome if we did it this way. Right now in our config file we just give it 1 prefix and we only need to manage 1 config file per account. If we use this schema `/environments/%%ENVIRONMENT%%/global/user-managment` we would need to have 3 different config file per enviroment